### PR TITLE
cmd/snap-update-ns: add better unit test for overname sorting

### PR DESCRIPTION
**Pull request by bboozzoo**
A followup to #9751.

Add a simpler test case for checking the order of sorting with overname.

### CODE REVIEW

File: cmd/snap-update-ns/sorting_test.go [hunk 1]
```diff
@@ -73,6 +73,25 @@ func (s *sortSuite) TestParallelInstancesAndSimple(c *C) {
...
 
+func (s *sortSuite) TestOvernameOrder(c *C) {
+	expected := []osutil.MountEntry{
+		{Dir: "/a/b/2", Options: []string{osutil.XSnapdOriginOvername()}},
+		{Dir: "/a/b"},
+	}
+	entries := []osutil.MountEntry{
+		{Dir: "/a/b"},
+		{Dir: "/a/b/2", Options: []string{osutil.XSnapdOriginOvername()}},
+	}
```
Comment (stolowski) [inline cmd/snap-update-ns/sorting_test.go:84]: Would it make sense to have a 3rd entry that is unrelated to the two key entries?

**Review by stolowski — APPROVED** [review]
LGTM, +1

File: cmd/snap-update-ns/sorting_test.go (hunk 1 above)
Comment (bboozzoo) [reply cmd/snap-update-ns/sorting_test.go:84]: No, the point is to have a minimal set of entries such that the test cases of ith(overname) < jth, and ith > jth(overname) are executed. The test cases with more mixed entries are covered by other unit tests (TestParallelInstancesAndSimple and TestParallelInstancesAlmostSorted)

Comment (stolowski) [reply cmd/snap-update-ns/sorting_test.go:84]: Got it, thanks!

**Review by mvo5 — APPROVED** [review]
LGTM
